src/pages/SubmissionHistory/SubmissionHistory.test.js

```diff
@@ -349,22 +343,6 @@ describe('SubmissionHistory page', () => {
 
     expect(screen.getByText('Some submission statuses are not up to date. Refreshing the page may update more statuses.')).toBeInTheDocument()
   })
-})
-
-describe('Actions column', () => {
-  beforeEach(() => {
-    getConfig.mockReturnValue({ isActionMenuEnabled: true })
-  })
-
-  it('Shows Actions column on table', async () => {
```

Comment:
This looks like it should still be valid? In fact... even more valid now, since it should _always_ be shown?

Reply:
I figured the other tests are now looking for 6 columns, so that should cover checking for its existence. No need for a specific test.